docs/ref/forms/models.txt

```diff
@@ -1,15 +1,258 @@
-====================
-Model Form Functions
-====================
-
-Model Form API reference. For introductory material about model forms, see the
-:doc:`/topics/forms/modelforms` topic guide.
+===========
+ModelForms
+===========
 
 .. module:: django.forms.models
-   :synopsis: Django's functions for building model forms and formsets.
+    :synopsis: ModelForm API reference for inner ``Meta`` class and factory
+        functions
+
+.. class:: ModelForm
+
+.. _ref-modelform:
+
+.. currentmodule:: django.forms
+
+:class:`~django.forms.ModelForm` API reference. For introductory material about
+ModelForms, see the :doc:`/topics/forms/modelforms` topic guide.
+
+.. _ref-modelform-meta-options:
+
+ModelForm ``Meta`` Options
+==========================
+
+The :class:`~django.forms.ModelForm` helper class can be used to build forms
+that closely reflect specific Django models. The exact structure of the form
+produced can be controlled by setting metadata options as attributes of an inner
+``Meta`` class.
+
+::
+
+    # Example using some sort of Book model
+
+    from django.forms import ModelForm
+    from myLibraryApp.models import Book
+
+
+    class BookForm(ModelForm):
+        class Meta:
+            model = Book
+            fields = ["title", "author"]
+            help_texts = {
+                "title": "The title of the book",
+                "author": "The author of the book",
+            }
+            # ... other attributes
+
+.. admonition:: Invalid Meta class attributes
+
+    While any number of attributes can be included in a ``Meta``
+    class, **only attributes that match those described below will be used**.
+    When troubleshooting or encountering unexpected behavior, be sure to check
+    for typos or misnamed attributes because these will **not**, in and of
+    themselves, raise an error (though they might cause another error to be
+    raised if they affect a required attribute).
+
+Other than :attr:`~ModelFormOptions.model` and at least one of
+:attr:`~ModelFormOptions.fields` and :attr:`~ModelFormOptions.exclude`, ``Meta``
+attributes are optional. It is **not** necessary for every option to be set for
```

Comment:
I think this can be improved if we list things in a more definitive manner such as:

```suggestion
:attr:`~ModelFormOptions.model` is required as well as either 
:attr:`~ModelFormOptions.fields` or :attr:`~ModelFormOptions.exclude`.
All other ``Meta`` attributes are optional.
```

I think this is clearer because it indicates the required state and the fields immediately rather than listing things, then indicating that this list is required, while everything else is optional.

I don't think this is a required change.